.ci/operatorhub.sh

```diff
@@ -14,6 +14,7 @@ OLD_PWD=$(pwd)
 VERSION=$(grep operator= versions.txt | awk -F= '{print $2}')
 PKG_FILE=deploy/olm-catalog/jaeger-operator/jaeger-operator.package.yaml
 CSV_FILE=deploy/olm-catalog/jaeger-operator/${VERSION}/jaeger-operator.v${VERSION}.clusterserviceversion.yaml
+CRD_FILE=deploy/crds/jaegertracing.io_jaegers_crd.yaml
```

Comment:
Currently there are two `_crd.yaml` files in that folder, one with the `v1` version in the name. Which one should be copied? Maybe we need to standardise on a naming convention and use it in the manual deploy instructions aswell - i.e. we might have multiple versions at some point - are they all going to be included in the same (non versioned) crd file, or in separate files?

Reply:
I just tried creating a new operator, and newer versions of the operator-sdk generate a file following the pattern without the version, with a pluralized kind name (`jaegertracing.io_jaegers_crd.yaml`). We should be able to remove `jaegertracing_v1_jaeger_crd.yaml` then.